resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -105,20 +105,44 @@ public DnsNameResolverBuilder eventLoop(EventLoop eventLoop) {
         return this;
     }
 
+    /**
+     * Use {@link #datagramChannelFactory()}
+     */
+    @Deprecated
     protected ChannelFactory<? extends DatagramChannel> channelFactory() {
-        return this.channelFactory;
+        return this.datagramChannelFactory;
+    }
+
+    ChannelFactory<? extends DatagramChannel> datagramChannelFactory() {
+        return this.datagramChannelFactory;
     }
 
     /**
      * Sets the {@link ChannelFactory} that will create a {@link DatagramChannel}.
      * If <a href="https://tools.ietf.org/html/rfc7766">TCP fallback</a> should be supported as well it is required
      * to call the {@link #socketChannelFactory(ChannelFactory) or {@link #socketChannelType(Class)}} method.
      *
-     * @param channelFactory the {@link ChannelFactory}
+     * @param datagramChannelFactory the {@link ChannelFactory}
      * @return {@code this}
+     * @deprecated use {@link #datagramChannelFactory(ChannelFactory)}
      */
-    public DnsNameResolverBuilder channelFactory(ChannelFactory<? extends DatagramChannel> channelFactory) {
-        this.channelFactory = channelFactory;
+    @Deprecated
+    public DnsNameResolverBuilder channelFactory(ChannelFactory<? extends DatagramChannel> datagramChannelFactory) {
+        datagramChannelFactory(datagramChannelFactory);
+        return this;
+    }
+
+    /**
+     * Sets the {@link ChannelFactory} that will create a {@link DatagramChannel}.
+     * If <a href="https://tools.ietf.org/html/rfc7766">TCP fallback</a> should be supported as well it is required
```

Comment:
Consider adding `<p>` in between lines 136 and 137 to separate initial line with other clarifications